.travis.yml

```diff
@@ -17,27 +11,14 @@ addons:
 env:
   - VIBED_DRIVER=libevent
   - VIBED_DRIVER=libev
-  - VIBED_DRIVER=libasync
-
-matrix:
-  exclude:
-    - d: dmd-2.065.0
-      env: VIBED_DRIVER=libasync
-    - d: ldc-0.14.0
-      env: VIBED_DRIVER=libasync
-    - d: ldc-0.15.1
-      env: VIBED_DRIVER=libasync
-    - d: gdc-4.9.0
-      env: VIBED_DRIVER=libasync
-    - d: gdc-4.9.2
-      env: VIBED_DRIVER=libasync
+  - VIBED_DRIVER=libasync_botan
 
 script:
   - dub test --compiler=$DC --config=${VIBED_DRIVER} || exit 1
-  - if [ ${VIBED_DRIVER} == "libevent" ]; then
+  - if [ ${VIBED_DRIVER} == "libasync_botan" ]; then
       for ex in `\ls -1 examples/`; do (echo "[INFO] Building example $ex"; cd examples/$ex && dub build --compiler=$DC) || exit 1; done
     fi
-  - if [ ${VIBED_DRIVER} == "libevent" ]; then
+  - if [ ${VIBED_DRIVER} == "libasync_botan" ]; then
```

Comment:
I'll submit a change to separate the driver and whether or not it does the full testing when I get home.